### PR TITLE
Added a bundle deprecation notice in other articles

### PR DESCRIPTION
**Pull request by javiereguiluz**
Similar to #8389 but for another article.

### CODE REVIEW

**Review by xabbuh — APPROVED** [review]

**Comment by wouterj** [comment]
:-1: This article is exactly explaining how to override stuff **without bundle inheritance**. So this article becomes more important now inheritance is deprecated. It makes no sense to add this caution there imo.

**Comment by javiereguiluz** [comment]
This article may not be about bundle inheritance, but it mentions that four times:

inheritance-1

inheritance-2

**Comment by wouterj** [comment]
Then, instead of adding this confusing warning, I think we should remove the references to the other article in this article.

File: bundles/override.rst [hunk 1]
```diff
@@ -5,7 +5,8 @@ How to Override any Part of a Bundle
 ====================================
 
 This document is a quick reference for how to override different parts of
-third-party bundles.
+third-party bundleswithout using :doc:`/bundles/inheritance`, which is deprecated
```
Comment (xabbuh) [inline bundles/override.rst]: missing space between "bundles" and "without"

**Review by xabbuh — APPROVED** [review]

**Review by wouterj — APPROVED** [review]

**Comment by xabbuh** [comment]
Thank you Javier.

**Pull request closed without merging**